### PR TITLE
Python: Version bump to 3.11.2

### PR DESCRIPTION
**Pull request by dagbrown**

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: python/python/BUILD [hunk 1]
```diff
@@ -2,6 +2,7 @@ OPTS+=" --enable-shared \
         --enable-ipv6 \
         --with-system-ffi \
         --with-system-expat \
+        --with-system-libmpdec \
```
Comment (Ratler) [inline python/python/BUILD:5]: This is already an optional dependency. Is it really a mandatory dependency?

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by esselfe** [comment]
Compiled and installed fine. The reinstallation of the python modules is a bit tricky though.
Some of the modules rebuilt failed making me think a list to be recompiled in POST_INSTALL after should do the trick.

Like python-setuptools, meson, python-build, python-packaging, python-six and some others.

Python 3.11 has also been tested on my Sourcemage system without problems.

**Comment by esselfe** [comment]
There might just be protobuf which fails with:

google/protobuf/pyext/descriptor.cc:135:32: error: invalid use of incomplete type 'PyFrameObject' {aka 'struct _frame'}
  135 |   if (frame->f_globals != frame->f_locals) {
      |                                ^~
/usr/include/python3.11/pytypedefs.h:22:16: note: forward declaration of 'PyFrameObject' {aka 'struct _frame'}
   22 | typedef struct _frame PyFrameObject;

and similar ones.

**Pull request merged**